src/mcts/stoppers/stoppers.cc

```diff
@@ -211,6 +213,7 @@ bool SmartPruningStopper::ShouldStop(const IterationStats& stats,
   // May overflow if (nps/smart_pruning_factor) > 180 000 000, but that's not
   // very realistic.
   hints->UpdateEstimatedRemainingRemainingPlayouts(remaining_playouts);
+  if (stats.batches_since_movestart < minimum_batches_) return false;
```

Comment:
Maybe also log batches_since_movestat in the log message below (line 230), so that we notice if e.g. it instamoves as soon as number of required batches comes.

Reply:
Sounds useful for setting a value that is high enough to do something yes.